src/components/Curations/DashboardContainer/DashboardContainer.js

```diff
@@ -1,16 +1,42 @@
+'use client'
+
 import './DashboardContainer.css'
-import curationsData from '/src/mock/curations.json'
 import Carousel from '../Carousel/Carousel'
+import React, { useState, useEffect } from 'react'
```

Comment:
```suggestion
import { useState, useEffect } from 'react'
```

하나 까먹었는데 `import React from "react"`는 파일에 굳이 쓸 필요가 없는 구문이지...만, React 17 이전까지는 반드시 필요했기에 이전 튜토리얼이나 코드 자동완성을 하다 보면 포함될 수도 있다! 지워주는 게 좋음

관련해서는 [You no longer need to import React from "react"](https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj)나 [관련 글](https://so-so.dev/react/import-react-from-react/)을 참고하면 좋다. React 17이 릴리즈된 게 2020년 즈음인데(지금은 react 18), 이렇게 대규모 라이브러리의 변천 과정을 보는 것도 나름의 묘미